DSCResources/MSFT_xCertificateImport/MSFT_xCertificateImport.schema.mof

```diff
@@ -0,0 +1,8 @@
+[ClassVersion("1.0"), FriendlyName("xCertificateImport")]
+class MSFT_xCertificateImport : OMI_BaseResource
+{
+[Key] string Thumbprint;
+[Key] string Path;
+[write,ValueMap{"LocalMachine"},Values{"LocalMachine"}] string Location;
+[write] string Store;
```

Comment:
Are the keys correct?  The key should uniquely identify the resource you are affecting.  For example, in the package resource, the path of the package you are installing is not part of the key.  The key is the ID, and name of the package.